js/tests/unit/modal.spec.js

```diff
@@ -11,8 +11,11 @@ describe('Modal', () => {
   beforeAll(() => {
     fixtureEl = getFixture()
 
-    // Enable the scrollbar measurer
-    const css = '.modal-scrollbar-measure { position: absolute; top: -9999px; width: 50px; height: 50px; overflow: scroll; }'
+    // Enable the scrollbar measurer and reset body margin
+    const css = `
+      .modal-scrollbar-measure { position: absolute; top: -9999px; width: 50px; height: 50px; overflow: scroll; }
+      body { margin: 0; }
```

Comment:
While not strictly necessary for the test to work, I believe it makes sense to include this style definition here: The values of `rect.left` and `rect.right` in the `_checkScrollbar` method depend on this margin, and when not set explicitly, a browser default will be used (e.g. `8px` in Chrome) instead of the [value actually set by Bootstrap](https://github.com/twbs/bootstrap/blob/master/scss/_reboot.scss#L41).